src/openforms/templates/maykin_2fa/login.html

```diff
@@ -1,11 +1,14 @@
 {% extends "maykin_2fa/login.html" %}
-{% load solo_tags i18n %}
+{% load solo_tags i18n static %}
 
 {% block extra_login_options %}
     {% get_solo 'mozilla_django_oidc_db.OpenIDConnectConfig' as oidc_config %}
     {% if oidc_config.enabled %}
-        <div class="admin-login-option">{% trans "or" %}</div>
-        <div class="admin-login-option">
+        <div class="admin-login-option admin-login-password">
+            <a href="#">{% trans "Login with application account" %}</a>
```

Comment:
I think a `<button>` element is semantically more correct, but I'm not sure if there are easy CSS classes available that make it look like a link. Either way, you want to call `event.preventDefault()` with links so that the `#` doesn't end up in the URL.

Reply:
I'll keep it as a link and add `event.preventDefault` to the handler